fish.xcodeproj/project.pbxproj

```diff
@@ -679,15 +680,18 @@
 				D04F7FAE1BA4DDB500B0F227 /* pcre2_serialize.c */,
 				D04F7FB11BA4DDBF00B0F227 /* pcre2_string_utils.c */,
 				D04F7FB41BA4DDC900B0F227 /* pcre2_study.c */,
-				D04F7FB71BA4DDEB00B0F227 /* pcre2_substitute.c */,
 				D04F7FB81BA4DDEB00B0F227 /* pcre2_substring.c */,
+				9CCD21741CC2280600FC97C8 /* pcre2_auto_possess.c */,
 				D04F7FB91BA4DDEB00B0F227 /* pcre2_tables.c */,
 				D04F7FBA1BA4DDEB00B0F227 /* pcre2_ucd.c */,
 				D04F7FBB1BA4DDEB00B0F227 /* pcre2_valid_utf.c */,
+				9C40B6F81CC225E800D2751F /* pcre2_substitute.c */,
+				9C40B6F91CC225E800D2751F /* pcre2_compile.c */,
 				D04F7FBC1BA4DDEB00B0F227 /* pcre2_xclass.c */,
 				D04F7FF71BA4E82C00B0F227 /* pcre2_chartables.c.dist */,
 			);
 			name = pcre;
+			path = "../pcre2-10.21";
```

Comment:
I set a path on the pcre group/folder in Xcode, and the paths for the sources are relative to that - in the future it will only be necessary to change this once, on that, for bumps, instead of having to change the reference for each individual .c file.